src/screens/PinFlow/CurrentPinScreen.tsx

```diff
@@ -61,41 +60,44 @@ class CurrentPinScreen extends PureComponent<Props, State> {
     currentDate = currentDate.add(blockedTimeInMinutes, 'minute');
 
     if (isFinalAttempt) {
-      this.props.setTimeCounter(currentDate.unix());
-      this.props.setFailedAttempts(attempt + 1);
+      setTimeCounter(currentDate.unix());
+      setFailedAttempts(attempt + 1);
+      setFailedAttemptStep(0);
       this.setState({ isCount: true });
+    } else {
+      setFailedAttemptStep(increasedFailedAttemptStep);
     }
 
-    return increasedFailedTimes !== firstAttempt && increasedFailedTimes !== finalAttempt
-      ? `\n${i18n.onboarding.failedTimesErrorInfo} ${blockedTimeInMinutes} ${i18n.onboarding.minutes}\n${i18n.onboarding.failedTimes} ${increasedFailedTimes}/${finalAttempt}`
+    return increasedFailedAttemptStep !== firstAttempt && increasedFailedAttemptStep !== finalAttempt
```

Comment:
will be there a situation when increasedFailedAttemptStep could be equal 0? so is the condition `increasedFailedAttemptStep !== firstAttempt` needed?

Reply:
sure, not needed :+1: